### DISCUSSION TITLE
Disable a Device

### DISCUSSION THREAD

**Discussion by Eistee82** [Koenkk/zigbee2mqtt#15132] [General]
Hi I have some Devices wich are not always powered like vernilators or klimate devices. In winter time I get many errors of not responding Devices. Is there a way to Disable them so z2m is no longer searching for this devices?

**Comment by mjhcdk** [comment]
I have same wish, being able to disable some season devices and other LAB devices without having to delete them.

**Comment by Koenkk** [comment]
https://github.com/Koenkk/zigbee2mqtt/issues/10003

**Reply by marvin78** [reply]
Will it be possible to disable a device through MQTT?

**Reply by Koenkk** [reply]
Yes, publish to topic `zigbee2mqtt/bridge/request/device/options` payload `{"id": "DEVICE_FRIENDLY_NAME","options":{"disabled": true}}`, next publish and empty payload to `zigbee2mqtt/bridge/request/restart` to restart z2m.

**Reply by marvin78** [reply]
Ah Thanks!
